contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -31,6 +31,26 @@ end
 $$
 LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION sys.babelfish_update_server_collation_name() RETURNS VOID
```

Comment:
I am not sure why is this required?

Reply:
Same reason as above. We need to put update the collation info for all such versions considering the user is on 15.1, 15.2 and so on. Only one upgrade script is not sufficient to address the issue.